src/pylibsshext/channel.pyx

```diff
@@ -105,6 +112,25 @@ cdef class Channel:
     def sendall(self, data):
         return self.write(data)
 
+    def read_bulk_response(self, size=1024, stderr=0, timeout=0.001, retry=5):
+        recv = BytesIO()
```

Comment:
It's better to always use a context manager because currently, you'll have an unclosed resource if any exception happens in the middle of the loop below.